examples/medplum-provider/src/App.tsx

```diff
@@ -122,7 +123,7 @@ export function App(): JSX.Element | null {
               <Route path="/" element={<HomePage />} />
               <Route path="/Patient/:patientId" element={<PatientPage />}>
                 <Route path="Encounter/new" element={<EncounterModal />} />
-                <Route path="Encounter/:encounterId" element={<EncounterChart />}>
+                <Route path="Encounter/:encounterId" element={<EncounterChart key={location.pathname} />}>
```

Comment:
Can we push this `key={location.pathname}` into the `<EncounterChart>` component?  We try to keep logic out of the route map.